cmd/collector/app/handler.go

```diff
@@ -80,7 +87,45 @@ func (aH *APIHandler) saveSpan(w http.ResponseWriter, r *http.Request) {
 		} else {
 			w.WriteHeader(http.StatusOK)
 		}
+
+	case "zipkin.thrift":
+		spans, err := deserializeZipkin(bodyBytes)
+		if err != nil {
+			http.Error(w, fmt.Sprintf(unableToReadBodyErrFormat, err), http.StatusBadRequest)
+			return
+		}
+
+		ctx, _ := tchanThrift.NewContext(time.Minute)
+		if _, err = aH.zipkinSpansHandler.SubmitZipkinBatch(ctx, spans); err != nil {
+			http.Error(w, fmt.Sprintf("Cannot submit Zipkin batch due to error: %v", err), http.StatusInternalServerError)
+			return
+		}
+
+		w.WriteHeader(http.StatusOK)
```

Comment:
nit: add `return` to `default:` clause, and move OK status to the end, since it is repeated